Dredd announcement

honzajavorek opened the pull request:
Waits for @netmilk / @zdne to give it a go, but anyone else is also welcome to add any comments and thoughts.

kylef commented on _posts/2016-07-04-Test-Your-Swagger-API.md:
```diff
@@ -0,0 +1,47 @@
+
+Expressive power of API description formats has its limits and testing can easily get rather complex when it meets real-world situations. One of the coolest features of Dredd is its extensivity by so-called [hooks][]. Hooks are pieces of arbitrary code you can get executed before or after every test case.
+
+Originally Dredd was able to execute only hooks written in JavaScript, but in recent months it got first-class support for many other languages, namely Perl, PHP, Python, Ruby, and Go (experimental). Since there's a [tutorial][hooks-new-language] how to implement new hook handler, every now and then contributors come with support for their favorite language.
```
I'd suggest making `Perl, PHP, Python, Ruby, and Go` individual hyperlinks to the appropriate hook documentation. I'm sure people reading this might already be using dredd and want to hear how to use their favourite language with dredd.

Almad commented on _posts/2016-07-04-Test-Your-Swagger-API.md:
```diff
@@ -0,0 +1,53 @@
+
+It's a [long time][original-dredd-article] since we announced [Dredd][dredd], an Open Source framework for testing HTTP-based APIs. Over the years it grown into a fully-featured, mature, and very popular tool. The main flaw remaining was the fact that Dredd was able to read only [API Blueprint][api-blueprint] format.
+
+Now that's not true anymore! With 1.1.0 release, all Dredd's features are now available also for users of the [Swagger][swagger] API description format. Since version 1.5.0 the Swagger support is considered production-ready. Here we would like to thank to all the adventurous minds who helped us to improve the initial implementation. Only thanks to you we could remove the "experimental" label so s ...
```
nit: I'd cross-link versions to releases

zdne replied:
Remove cruft: 

"At Apiary we're proponents of design..."

zdne replied:
Change the tone 

> The main flaw remaining

it wasn't a flaw. that was the design of the tool. Turn it around. E.g until now it supported only API Blueprint, but we want empower everybody to test their API regardless of the chosen description format..

Almad commented:
@honzajavorek I'll leave English linting to others (I'd change some expressions a bit, but I am not that confident about it). 

I'd make more fuzz / add more emphasis on our call for contributors, I'd make that one of the three main points. I think it would also be great to have the major points from the changelog as an item with links to issues / documentations. 

I also think that it'd be great to reuse visuals made for twitter announcements, it usually make articles more readable.

But otherwise, great work! Looking forward to have it out :)

zdne commented:
> I also think that it'd be great to reuse visuals made for twitter announcements, it usually make articles more readable.

We will do the visuals, @honzajavorek do not worry about it.

zdne commented:
👍 with me. @honzajavorek can you please fix my two comments and we are good to go?

honzajavorek commented:
@Almad 

> I'd make more fuzz / add more emphasis on our call for contributors, I'd make that one of the three main points.

I tried to sneak in importance contributions as much as I could. I don't want to fragment the article into even more topics - it already contains a lot of information. Maybe dedicated article about contributing in the future.

> I think it would also be great to have the major points from the changelog as an item with links to issues / documentations.

I don't know. I want to keep this product-oriented and concrete points from changelog would feel to me as clutter here. Moreover, most of the work was under the hood, in Dredd Transactions, in transition from AST to API Elements, in Swagger adapter, etc.

honzajavorek commented:
@zdne Addressed comments, but CI is failing. Looks like either Circle has problems to reach http**s** version of apiblueprint.org or we have some SSL problems on the site, but all checks of links to https://apiblueprint.org blew up. I changed link in my article to http://apiblueprint.org and it seems to pass the check 😐

zdne commented:
@honzajavorek calling @JackuB to the rescue as I recall he was doing something with the links. Corvax, any clue what is wrong and how to fix it? Thanks!

kylef commented:
@honzajavorek We had problems with this in the past, we updated libcurl (https://github.com/apiaryio/apiaryio.github.com/pull/92/commits/5d586a0d305a56824724b1319567d92c1d0f90df) to support SNI solved the problem. Not sure why it's failing in this PR though. Does master also fail now?

w-vi commented:
` sudo apt-get install libcurl3-dev` is failing, that's the main issue I think. Leaving to Ubuntu experts to solve it.

honzajavorek commented:
@apiaryio/sre to the rescue?

abtris commented:
@honzajavorek fixed in https://github.com/apiaryio/apiaryio.github.com/pull/99

honzajavorek commented:
@abtris 💕 🔧 💕

@zdne ✨ 🚀 ✨

zdne commented:
Thanks everyone!
